docs/docs/ias.md

```diff
@@ -1,41 +1,49 @@
 # Intel Attestation Service
 
-!!! important
-    You don't need to read and perform these steps to deploy a Conclave app, unless you wish to use older
-    hardware that isn't capable of using the DCAP protocol. [Azure Gen 2 Virtual Machines](azure.md) in particular do not
-    need any of these steps.
+!!!Note
+    
+    You don't need to perform these steps if you use [Azure Gen 2 VMs](machine-setup.md) or hardware that supports the
+    DCAP attestation protocol.
 
-How do you know a remote computer is fully up to date with the latest security patches and isn't physically compromised?
-This is the question Intel's Attestation Service (IAS) exists to answer. IAS is an online service that evaluates
-evidence from an enclave and returns:
+Intel's Attestation Service (IAS) helps to verify that a remote computer isn't physically compromised and has the 
+latest security patches.
 
-1. The time at which the assessment was made
-2. Whether the system is considered secure or whether it needs:
+IAS evaluates evidence from an enclave and returns:
+
+1. The time at which the IAS made the assessment.
+2. If the system is secure or if it needs:
    1. Software upgrades
-   2. Operating system / microcode upgrades
+   2. OS or microcode upgrades
    3. BIOS configuration changes
-3. The CVE IDs of any active security advisories against the remote system
+3. The [CVE ID](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#CVE_identifiers)s of relevant
+security advisories.
 
-... along with various other pieces of data. The response is signed by Intel. Because it's signed it can be hosted and
-passed around by anyone, thus when an enclave is started in debug or production mode IAS is contacted and an attestation
-calculated. It's a part of a serialized [`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object and can be passed to clients from the server. This
-means clients don't need to contact Intel's servers themselves and thus don't need authorisation to use IAS.
+You can send and use the response anywhere, as Intel has signed it. When an enclave starts in debug or release mode, 
+it contacts the IAS to calculate the attestation. This attestation is a part of a serialized
+[`EnclaveInstanceInfo`](api/-conclave%20-core/com.r3.conclave.common/-enclave-instance-info/index.html) object and 
+can be passed to clients from the server. This means clients don't need authorization to use the IAS.
```

Comment:
I would add a `that`, what do you think? `This means that clients`

Reply:
Removed this sentence to address another review comment.